liveIT-db/schema/restaurants/Restaurants.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Restaurants (
+id INT AUTO_INCREMENT,
+name VARCHAR(200) NOT NULL,
+address VARCHAR(250) NOT NULL,
```

Comment:
address would take more than 250. should we have state column separately?

Reply:
we discussed we are not doing state separately.